github/data_source_github_repository_branches.go

```diff
@@ -0,0 +1,70 @@
+package github
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/go-github/v43/github"
```

Comment:
```suggestion
	"github.com/google/go-github/v45/github"
```
To match the currently used version of `go-github`.